### PR TITLE
Hooks

### PR DESCRIPTION
**Pull request by JPLozanoE**

### CODE REVIEW

**Comment by JPLozanoE** [comment]
De haber usado redux hubiera facilitado bastante el flujo de la info:p

File: src/components/Home/VideoItem.jsx [hunk 1]
```diff
@@ -15,9 +14,13 @@ const VideoText = styled.p`
 `;
 
 export const VideoItem = (props) => {
-  // console.log(props);
+  const { setSelectedVideo } = props;
+  const handleVideoClick = (video) => {
+    setSelectedVideo(video);
+    // console.log(video);
```
Comment (marrossa86) [inline src/components/Home/VideoItem.jsx:20]: Remember to remove the console logs when you are sending a PR.

**Review by marrossa86 — COMMENTED** [review]
Thanks JP,

The components looks ok. I think you are just missing the tests for all your new components.

File: src/components/Home/VideoItem.jsx (hunk 1 above)
Comment (JPLozanoE) [reply src/components/Home/VideoItem.jsx:20]: Sorry, got it!

Comment (JPLozanoE) [reply src/components/Home/VideoItem.jsx:20]: And for testing, had a session with another member of the bootcamp and finally figured out how to get the coverage, will work on testing these days 😁

**Pull request merged**